apps/evm/src/clients/api/queries/getPrimeToken/useGetPrimeToken.ts

```diff
@@ -30,14 +31,20 @@ type Options = QueryObserverOptions<
 const useGetPrimeToken = (input: UseGetPrimeTokenInput, options?: Partial<Options>) => {
   const { chainId } = useChainId();
   const isPrimeEnabled = useIsFeatureEnabled({ name: 'prime' });
-  const primeContract = useGetPrimeContract();
+  const primeContractAddress = useGetPrimeContractAddress();
+  const { publicClient } = usePublicClient();
 
   return useQuery({
     queryKey: [FunctionKey.GET_PRIME_TOKEN, { ...input, chainId }],
 
     queryFn: () =>
-      callOrThrow({ primeContract, accountAddress: input.accountAddress }, params =>
-        getPrimeToken(params),
+      callOrThrow(
+        {
+          primeContractAddress: primeContractAddress as Address,
+          accountAddress: input.accountAddress as Address,
```

Comment:
Aren't those already typed as `Address`?

Reply:
They are, that's an oversight on my end